packages/schema-blocks/src/core/blocks/BlockDefinition.tsx

```diff
@@ -89,4 +92,21 @@ export default class BlockDefinition extends Definition {
 		// Register the block with our own code.
 		registerBlockDefinition( name, this );
 	}
+
+	/**
+	 * Creates the sidebar elements.
+	 *
+	 * @param props The properties of the block to create a sidebar for.
+	 *
+	 * @returns {ReactElement[]} The sidebar element to render.
+	 */
+	sidebarElements( props: RenderEditProps ): ReactElement[] {
+		const sidebarElements: ReactElement[] = Object.values( this.instructions )
+			.map( ( instruction, index ) => {
+				const output = instruction.sidebar( props, index );
+				return output;
```

Comment:
`output` is returned immediately after its declaration, I'd just return the result of the function call without storing it to a variable
```suggestion
				return instruction.sidebar( props, index );
```

Reply:
the entire function is now a oneliner.